src/main/java/com/ezasm/gui/RegisterTable.java

```diff
@@ -42,6 +44,26 @@ public RegisterTable(Registers registers) {
         add(scrollPane);
     }
 
+    /**
+     *      * The function to highlight changed registers      
```

Comment:
This javadoc comment is improperly formatted and contains characters we don't use -- `NBSP`

Reply:
I believe this is changed by the maven formatter

Reply:
I'll look into it